### DISCUSSION TITLE
Does renovate support HELM version in AWS-CDK?

### DISCUSSION THREAD

**Discussion by gkaskonas** [renovatebot/renovate#21578] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.99.2

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Ability to update helm version inside .ts files when using aws cdk
```
export function configureRedisChart(cluster: ICluster): HelmChart {
  return cluster.addHelmChart(serviceName, {
    chart: serviceName,
    createNamespace: false,
    namespace: serviceName,
    repository: "https://charts.bitnami.com/bitnami",
    release: serviceName,
    version: "17.4.2",
```

I want renovate to update the repo when a new version is available

### What you tried so far.

Nothing

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by gkaskonas** [comment]
This looks related https://github.com/renovatebot/renovate/discussions/10299 but its not clear what was required to make it work?

**Comment by rarkins** [comment]
This looks like something best done with a regex manager
